### ISSUE TITLE
Settings dialog tab behavior different on first open

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#534]
- *Please use a short, but meaningful title.*
- *Delete everything in italics before submitting issue*
- *Make sure to fill in as much information as possible*

#### Expected Behavior
Style, Layout, and Keyboard tabs within the settings dialog should only be navigated via the arrow keys.  Only the currently selected tab should be in the tab order.

#### Observed behavior
The first time the settings dialog is opened all of the tabs are in the tab order until they are activated/selected once. 

#### Steps to reproduce
1.Load Readium (with either library or reader view)
2.Invoke the Settings dialog using the keyboard. Focus is on the Close icon
3. press tab - focus goes to the style tab
4. press tab - focus goes to the Layout tab - it should go the the first element in the style panel
5. press tab again - focus goes to the Keyboard tab 
6. press tab again - focus is now on the  font size adjustment in the Style tab
7. The Layout tab and Keyboard tabs will stay in the tab focus ring until they are selected.

Once the Layout and Keyboard tabs have been selected once, then the tab panel navigation works properly with the arrow keys and only the last selected tab remains in the tab order.

This is likely just a misuse of the tabindex attribute but needs further investigation to see if it can be fixed within Readium or is an issue with the underlying Bootstrap dialog code. 

### Product
* Readium cloud reader app
  Wed, 15 Jun 2016 15:16:04 GMT
readium-js-viewer@39939d854bc27f945f26b9ffa265bdff8ccee35c
readium-js@b32f9ebc23d98dd23bdbee9d485d4ad9e736afc1
readium-shared-js@91324db9ca2cb3b931761a115384f2328737103f
readium-cfi-js@a9b07a78fa73215c624cdeb2047ab6dc49f81a7d
  * Browser and version
Safari Version 9.1.1 (11601.6.17)
  * OS and version
OS X 10.11.5

#### Additional information
*Please list any additional information that might help us*

**Comment by danielweck** [comment]
Fixed by https://github.com/readium/readium-js-viewer/pull/535
